<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetProperty</class>
 <widget class="QWidget" name="NetProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>网络接口：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="com_Interface"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>7</number>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>属性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>接口索引</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>最大传输单元</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>接口名称</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>关联标志</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>接口类型</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>物理地址</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="textEdit_ip"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
